README.md

```diff
@@ -89,6 +89,8 @@ If you have questions or comments, please create an issue.
 - [Open Source with Pradumna](https://github.com/Pradumnasaraf/open-source-with-pradumna) - This repo contains resources and materials to learn and get yourself started with Open Source, Git, and GitHub.
 - ["FOSS Community Acronyms"](https://github.com/d-edge/foss-acronyms) - This repo contains a list of abbreviations used within the FOSS (Free and Open Source) community, along with their definitions and usages.
 - ["Open Source Fiesta - Open Source Fiesta"](https://zubi.gitbook.io/open-source-fiesta/) - Step-by-step instruction on how to contribute to GitHub repositories, and includes a git command line cheatsheet.
+- ["Beginner's Guide to Open Source Contribution - by Love Babbar Youtube Channel"](https://www.youtube.com/watch?v=SXW_bA4SGqA) - A hindi video tutorial for beginners to understand what is Open Source.
+- ["How to Fix your First Bug in OPEN SOURCE - by Love Babbar Youtube Channel"](https://www.youtube.com/watch?v=fMdfDnGX9S8) - A hindi video tutorial for beginners to fix your first bug in Open Source.
```

Comment:
Thanks for sharing these! Because these are in Hindi, I think removing these and adding them to the Hindi README file (https://github.com/freeCodeCamp/how-to-contribute-to-open-source/blob/main/README-HI.md) would be more appropriate.

Reply:
I'm sorry but most of the Hindi speaking people prefer to read in English. So they're going to see the English readme file and open the link to the Hindi tutorial. So I think it's appropriate.